interchaintest/upgrade_noble-1_test.go

```diff
@@ -27,16 +27,16 @@ func TestNoble1ChainUpgrade(t *testing.T) {
 		{
 			upgradeName: "radon",
 			image:       ghcrImage("v3.0.0"),
+			postUpgrade: testPostRadonUpgrade,
 		},
 		{
 			upgradeName: "v3.1.0",
 			image:       ghcrImage("v3.1.0"),
-			postUpgrade: testPostRadonUpgrade,
 		},
 		{
 			upgradeName: "argon",
 			image:       nobleImageInfo[0],
-			// postUpgrade: testPostArgonUpgradeMainnet,
+			postUpgrade: testPostArgonUpgrade,
```

Comment:
Unfortunately, we are going to have to remove the post-upgrade part from the mainnet upgrade test if we want this to pass.

This is because of the way we created the fiat-tokenfactory where you must have an owner in state. IE, you have to hard code the owner role in the `neon` upgrade handler.

The reason this works in the testnet upgrade test is because we altered the fiat-tokenfactory owner in this image of noble:
 https://github.com/strangelove-ventures/noble/blob/72e36f89a65376d40a339cb9cf8b99fc461628cf/interchaintest/upgrade_grand-1_test.go#L25

The owner in the image above, matches the owner here:
https://github.com/strangelove-ventures/noble/blob/72e36f89a65376d40a339cb9cf8b99fc461628cf/interchaintest/upgrade_argon_test.go#L38